Hardcore.lua

```diff
@@ -1336,6 +1341,24 @@ function Hardcore:InitiatePulseCheck()
 	end)
 end
 
+function Hardcore:InitiatePulsePlayed()
+	--init time played
+	Hardcore:RequestTimePlayed()
+
+	--time accumulator
+	C_Timer.NewTicker(TIME_TRACK_PULSE, function()
+		Hardcore_Character.time_tracked = Hardcore_Character.time_tracked + TIME_TRACK_PULSE
```

Comment:
Having this here as opposed to in the requestplayed timer will offset the accumulation by just a bit; but thats probably okay. As long as time played pulse doesn't approach our time gap limit